src/components/DeploymentsCount.vue

```diff
@@ -35,4 +32,10 @@
 
   const deploymentsCountSubscription = useSubscriptionWithDependencies(api.deployments.getDeploymentsCount, deploymentsCountFilter)
   const deploymentsCount = computed(() => deploymentsCountSubscription.response ?? null)
```

Comment:
While you're here, there's a `useDeploymentsCount` composition that would simplify this quite a bit. 
```typescript
const { count } = useDeploymentsCount(() => ({
  flows: {
    id: [props.flowId]
  }
})
```